embark.el

```diff
@@ -879,6 +900,8 @@ minibuffer before executing the action."
                            (not (memq action embark-skip-edit-commands))
                          (memq action embark-allow-edit-commands)))
            (inject
+            ;; TODO consider using strings for regions,
```

Comment:
I'd like to use strings. The only issue are some actions like write-region that prompt for a file name or something and would have the region contents incorrectly injected. There aren't that many of those, I think, so we just need to wrap them. An `embark--ignore-target-command` macro would be good enough.

Reply:
Okay, let's just keep the TODO for now?

Reply:
Sure.